uint32 people_ID
uint32 width
uint32 height
int32 origin_x
int32 origin_y
geometry_msgs/Point32 points
uint32 depth_confidence
